README.rst

```diff
@@ -267,6 +267,23 @@ Quality Assurance checks
 This package contains some common QA checks that are used the
 automated builds of different OpenWISP modules.
 
+``openwisp-utils-qa-checks``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can do multiple `checkmigrations` by passing the arguments with space-delimited string.
+
+For example, this multiple `checkmigrations`::
```

Comment:
There was a small mistake here, it needs to be ``checkmigrations`` with two slant quotes.